packages/core/src/logger/SFPLogger.ts

```diff
@@ -55,6 +53,12 @@ export const COLOR_KEY_VALUE=chalk.black.bold.bgGreenBright;
 export default class SFPLogger {
   public static logLevel: LoggerLevel = LoggerLevel.INFO;
 
+  static setColorLevel(nocolor:boolean) {
+  if(nocolor)
+    chalk.level = 0
+  else 
+    chalk.level = 1;
```

Comment:
By default, Chalk will automatically the level of colour support based on the environment

Reply:
Hence, we don't need an else case